### PR TITLE
Day16-HangTran-Fix bug tabItem

### PR DESCRIPTION
**Pull request by at-hangtran**
#### Related issues (Optional)
Task #84

#### Description (Required)
- [x] Fix custom tablayout

### CODE REVIEW

File: app/src/main/res/drawable/bg_tab_item.xml [hunk 1]
```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="utf-8"?>
+<selector xmlns:android="http://schemas.android.com/apk/res/android">
+    <item android:drawable="@android:color/transparent" android:state_pressed="true" />
+</selector>
```
Comment (at-vinhhuynh) [inline app/src/main/res/drawable/bg_tab_item.xml]: add a break line

File: app/src/main/java/vn/asiantech/internship/tablayout/CustomTabLayout.java [hunk 2]
```diff
@@ -46,12 +46,12 @@ private void initStrokePaint() {
 
     public void onDraw(Canvas canvas) {
         if (mIsSelected) {
-            RectF rectFFill = new RectF(0, 0, getWidth(), getHeight() * 2);
-            RectF rectFStroke = new RectF(1, 1, getWidth() - 1, getHeight() * 2);
+            RectF rectFFill = new RectF(-1, 0, getWidth() + 1, getHeight() * 2 + 20);
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/tablayout/CustomTabLayout.java]: WARNING: [DrawAllocation] Avoid object allocations during draw/layout operations (preallocate and reuse instead)
 You should avoid allocating objects during a drawing or layout operation. These are called frequently, so a smooth UI can be interrupted by garbage collection pauses caused by the object allocations.

The way this is generally handled is to allocate the needed objects up front and to reuse them for each drawing operation.

Some methods allocate memory on your behalf (such as `Bitmap.create`), and these should be handled in the same way.

File: app/src/main/java/vn/asiantech/internship/tablayout/CustomTabLayout.java [hunk 3]
```diff
@@ -46,12 +46,12 @@ private void initStrokePaint() {
...
+            RectF rectFStroke = new RectF(-1, 1, getWidth() + 1, getHeight() * 2 + 20);
```
Comment (at-dev-support-01) [inline app/src/main/java/vn/asiantech/internship/tablayout/CustomTabLayout.java]: WARNING: [DrawAllocation] Avoid object allocations during draw/layout operations (preallocate and reuse instead)
 You should avoid allocating objects during a drawing or layout operation. These are called frequently, so a smooth UI can be interrupted by garbage collection pauses caused by the object allocations.

The way this is generally handled is to allocate the needed objects up front and to reuse them for each drawing operation.

Some methods allocate memory on your behalf (such as `Bitmap.create`), and these should be handled in the same way.

**Comment by at-vinhhuynh** [comment]
@at-tienhoang tien
I merge this PR, thank you!

**Pull request merged**